pkg/presto/util.go

```diff
@@ -77,6 +77,28 @@ func DropView(queryer db.Queryer, catalog, schema, viewName string, ignoreNotExi
 	return err
 }
 
+// QueryMetadata executes a "DESCRIBE" Presto query against an existing, fully-qualified
+// table name to determine that table's column information.
+func QueryMetadata(queryer db.Queryer, catalog, schema, tableName string) ([]Column, error) {
+	rows, err := ExecuteSelect(queryer, fmt.Sprintf("DESCRIBE %s", FullyQualifiedTableName(catalog, schema, tableName)))
+	if err != nil {
+		return nil, fmt.Errorf("failed to query the %s Presto table's metadata: %v", tableName, err)
+	}
+
```

Comment:
nit: the extra vertical spacing doesn't add much value and doesn't match with the surrounding code.